cms/management/commands/activation_reminder_mail.py

```diff
@@ -0,0 +1,79 @@
+#code to send automated reminder mails to registerd users for activating their account
+#to run this file please follow below instructions:
+#1. Go to project directory
+#2. run "python manage.py activation_reminder_mail.py >>reminder_mail_success_log.txt"
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Third Party Stuff
+from django.core.management.base import BaseCommand
+from django.db import transaction as tx
+from django.contrib.auth.models import User
+from cms.models import Profile
+from config import *
```

Comment:
There is no default email. We have something called administrator email and it is not no-reply